io.github.Adda0.Stardrop.metainfo.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.adda0.stardrop</id>
+
+  <name>StardropForked</name>
+  <summary>Unofficial fork of Stardrop with custom changes</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-3.0-or-later</project_license>
+
+  <description>
+    <p>
+      Stardrop is an open-source, cross-platform mod manager for the game Stardew Valley. It is built using the Avalonia UI framework.
+    </p>
+    <p>
+      Stardrop utilizes SMAPI (Stardew Modding API) to simplify the management and update checking for all applicable Stardew Valley mods. Stardrop is also non-destructive due to how it integrates with SMAPI, meaning it can be uninstalled without any impact on SMAPI, Stardew Valley or the user&amp;amp;apos;s original mod folder. Profiles are also supported, allowing users to have multiple mod groups for specific gameplay or multiplayer sessions. Stardrop also supports the Nexus Mods API, which allows for features such as endorsing mods directly from Stardrop. Other features, such as downloading mod updates via Stardrop are available to premium Nexus Mods users.
+    </p>
+  </description>
+  <content_rating type="oars-1.1" />
+  <launchable type="desktop-id">io.github.Adda0.Stardrop.desktop</launchable>
+  <screenshots>
+    <screenshot type="default">
+      <image>https://github.com/Adda0/stardrop-flatpak/blob/main/screenshots/main-screenshot-resized.png?raw=true</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/Adda0/stardrop-flatpak/blob/main/screenshots/updates-settings-screenshot-resized.png?raw=true</image>
+    </screenshot>
+    <screenshot>
+      <image>https://github.com/Adda0/stardrop-flatpak/blob/main/screenshots/search-author-mods-screenshot-resized.png?raw=true</image>
```

Comment:
Screenshots should be from a tag/commit

Reply:
Fixed. Linked them from a specific commit.